src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
-        valType: 'any',
-        dflt: {},
-        description: [
-            'Sets each group style.',
-            'For example, with `groups` set to *[\'a\', \'b\', \'a\', \'b\']*',
-            'and `style` set to *{ a: { marker: { color: \'red\' } }}',
-            'marker points in group *\'a\'* will be drawn in red.'
-        ].join(' ')
+        target: {
+            valType: 'any',
```

Comment:
not so weird:
```js
0.20000000000000001 === 0.2
true
```

We definitely want numbers treated as strings for these purposes, so `target: 1` should match `'1'` in the data, and vice versa.

`valType: 'string'` might make sense to help accomplish this - with `strict` omitted so numbers get stringified. That would cause us to ignore array, object, null, and undefined targets, which seems right. All else equal I might have permitted booleans to pass through (stringified) but if it simplifies things - which it looks like it does - then lets leave those out.